geode-core/src/main/java/org/apache/geode/cache/query/internal/ResultsCollectionPdxDeserializerWrapper.java

```diff
@@ -218,4 +218,12 @@ public void setElementType(ObjectType elementType) {
     results.setElementType(elementType);
   }
 
+  public String toString() {
+    String out = "size: " + size() + "\n";
+    Iterator iter = iterator();
+    while (iter.hasNext()) {
+      out += iter.next() + "\n";
+    }
+    return out;
+  }
```

Comment:
All instances of `\n` that you find should be changed to `System.lineSeparator()`.